tarynmartin/personal-portfolio
# Add content and styling for education component

#### What’s this PR do?
- add content and styling for education component
#### Where should the reviewer start?
- in Education.js and associated styles.js
#### How should this be manually tested?
- in terminal, run npm start
- in browser, click on name then click on graduation cap to see new content & styling
#### Any background context you want to provide?
#### What are the relevant tickets?
#8
